flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/functions/ScalarFunctions.scala

```diff
@@ -285,4 +296,15 @@ object ScalarFunctions {
     */
   def repeat(base: String, n: Int): String = StringUtils.repeat(base, n)
 
+  /**
+    * Returns a character corresponding to the input integer ASCII code.
+    */
+  def chr(ascii: Long): String = {
+    if (ascii == null || ascii < 0 || ascii > 255) {
```

Comment:
As far as I know, when the input exceeding the range (1 - 255), the behavior of the `chr()` function differs in some databases. It's fine to return a `null` value and maybe we should mention that in the documentation.